src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -739,4 +749,56 @@ private RepairRun buildRepairRunFromRow(Row repairRunResult, long id){
         .build(id);
   }
 
+  @Override
+  public boolean takeLeadOnSegment(long segmentId) {
+    Row lwtResult = session.execute(getLeadOnSegmentPrepStmt.bind(segmentId, ReaperApplication.reaperInstanceId, ReaperApplication.getInstanceAddress())).one();
+    if (lwtResult.getBool("[applied]")) {
+      LOG.debug("Took lead on segment {}", segmentId);
+      return true;
+    }
+    
+    // Another instance took the lead on the segment
+    LOG.debug("Could not take lead on segment {}", segmentId);
+    return false;
+  }
+
+  @Override
+  public boolean renewLeadOnSegment(long segmentId) {
+    Row lwtResult = session.execute(renewLeadOnSegmentPrepStmt.bind(ReaperApplication.reaperInstanceId, ReaperApplication.getInstanceAddress(), segmentId, ReaperApplication.reaperInstanceId)).one();
+    if (lwtResult.getBool("[applied]")) {
+      LOG.debug("Renewed lead on segment {}", segmentId);
+      return true;
+    }
+    return false;
+  }
+
+  @Override
+  public void releaseLeadOnSegment(long segmentId) {
+    Row lwtResult = session.execute(releaseLeadOnSegmentPrepStmt.bind(segmentId, ReaperApplication.reaperInstanceId)).one();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "lwtResult". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "lwtResult" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)